designer/server/src/lib/forms.js

```diff
@@ -56,6 +63,34 @@ export async function remove(id) {
   return Promise.resolve(true)
 }
 
+/**
+ * Get draft form definition
+ * @param {string} id
+ * @returns {Promise<import('@defra/forms-model').FormDefinition | null>}
+ */
+export async function getDraftFormDefinition(id) {
+  const endpoint = getDraftFormDefinitionEndpoint(id)
+  const getJsonByType = /** @type {typeof getJson<FormDefinition>} */ (getJson)
+  const { body } = await getJsonByType(endpoint)
+
+  return body
+}
+
+/**
+ * Update draft form definition
+ * @param {string} id
+ * @param {string} definition - form definition as string in JSON format
```

Comment:
I think we're supposed to can let Wreck send an object?

It'll set `Content-Type: application/json` automatically via `{ json: true }` in `postJson()`

```suggestion
 * @param {FormDefinition} definition - form definition as string in JSON format
```